regressionTests.cmake

```diff
@@ -1619,6 +1619,12 @@ add_test_compareECLFiles(CASENAME gsatprod
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR satellite)
+add_test_compareECLFiles(CASENAME network_01_wtest
```

Comment:
all the other NETWORK related is using TUNING, maybe this one can also. Unless you checked the results that without TUNING, it shows good comparison with and without the recent development related to NETWORK + WTEST. 

I only checked the results with TUNING, it looks good for regression.

Reply:
I suggest we add the test without TUNING. If it creates problems due to instabilites etc. we can add the TUNING later.

Reply:
Often when we add TUNING for the testing it is because the solution is instable wrt. small unrelated changes in the simulator. I assumed that was the reason for adding tuning for the network tests.